aws/validators_test.go

```diff
@@ -2,11 +2,12 @@ package aws
 
 import (
 	"fmt"
-	"github.com/aws/aws-sdk-go/service/cognitoidentity"
-	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
 	"regexp"
 	"strings"
 	"testing"
+
+	"github.com/aws/aws-sdk-go/service/cognitoidentity"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
```

Comment:
Hi @vincenting 👋  Can you please rebase this pull request? I'm guessing our recent removal of SDKv1 is causing the merge conflict here (the new path is `github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest` -- apologies it affected this contribution.